app/interfaces/controllers/user_auth_controller.go

```diff
@@ -0,0 +1,136 @@
+package controllers
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"project-symi-backend/app/interfaces/database"
+	"project-symi-backend/app/usecase"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+type UserAuthController struct {
+	Interactor usecase.UserAuthInteractor
+}
+
+func NewUserAuthController(sqlHandler database.SqlHandler) *UserAuthController {
+	return &UserAuthController{
+		Interactor: usecase.UserAuthInteractor{
+			UserAuthRepository: &database.UserAuthRepository{
+				SqlHandler: sqlHandler,
+			},
+		},
+	}
+}
+
+func (controller *UserAuthController) LoginUser(c Context) {
+	//READ HEADER INTO A CREDENTIALS STRUCT
+	type UserCredentials struct {
+		EmployeeId string `json:"userId"`
+		Pass       string `json:"password"`
+	}
+	var user UserCredentials
+	if err := c.BindJSON(&user); err != nil {
+		panic(err)
+	}
+
+	//CHECK IF PASSED CREDENTIALS MATCH
+	tokenId, permissionLevel, err := controller.Interactor.CheckUserPass(user.EmployeeId, user.Pass)
+	if err != nil {
+		c.JSON(401, NewError(err))
+		return
+	}
+
+	//GET SECRET SIGNING PASSWORD
+	signingKey := []byte(os.Getenv("SIGNING_KEY"))
+
+	//GENERATE JWT FROM TOKEN ID
+	token := jwt.New(jwt.SigningMethodHS256)
```

Comment:
I think the best practice is to create jwt handler in infrastructure. But it's MVP.That's fine to me.
If you move it, this controller becomes slim.

Reply:
As discussed, for now moved the whole thing to a separate function.
Will create a separate handler for it during the refactoring stage